registry/remote/repository.go

```diff
@@ -102,6 +102,13 @@ type Repository struct {
 	// If less than or equal to zero, a default (currently 4MiB) is used.
 	MaxMetadataBytes int64
 
+	// SkipReferrersGC specifies whether to delete the dangling referrers index.
+	//  - If true, the old referrers index will be deleted after the new one
+	//    is successfully uploaded.
+	//  - If false, the old referrers index is kept.
+	// By default, it is enabled (set to true).
```

Comment:
It's better to have a reference link here.

Reply:
What link is expected to be put here? There is no spec mentioning how to do referrers index GC.

Reply:
https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#pushing-manifests-with-subject and https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#deleting-manifests？

Reply:
added as you suggested